server/package.json

```diff
@@ -72,6 +72,7 @@
     "request": "^2.88.0",
     "request-promise": "^4.2.5",
     "robots-parser": "^2.1.1",
+    "sane-email-validation": "^3.0.1",
```

Comment:
<img width="767" alt="Screenshot 2020-07-27 at 10 29 19 AM" src="https://user-images.githubusercontent.com/3964370/88505082-12a51500-cff4-11ea-953e-c5a1e615c072.png">

Does not look like a very reliable library to use, please change the library

Reply:
This is the only library which is helpful after researcching. All other libraries are either larger is size or has got no active contribution. Can we use this package itself ?

Reply:
Used 'vallidate.js' util, and made changes